src/arrays/programs_involving_multidimensional_data/easy/q4/Rotation.java

```diff
@@ -0,0 +1,33 @@
+package arrays.programs_involving_multidimensional_data.easy.q4;
+
+/*
+Write a Java program that initializes a 2D array of random integers of size 4 by 4, with the range of values from 1 - 99
+(inclusive of both values). Then, "rotate" the elements in the four corners of the array towards the clockwise direction
+using a method called rotateArray and print the result in the method. You should print the 2D array before and after the
+rotation.
+ */
+
+public class Rotation {
+    public static void main(String[] args) {
+        int[][] arr = new int[4][4];
+        System.out.println("Before the rotation:");
+        for (int i = 0; i < arr.length; i++)
+            for (int j = 0; j < arr[i].length; j++) {
```

Comment:
Change for loops to for more appropriate printing
`for (int i = 0; i < arr.length; i++) {
            for (int j = 0; j < arr[i].length; j++) {
                arr[i][j] = (int) (Math.random() * 99) + 1;
                System.out.print(arr[i][j] + " ");
            }
            System.out.println();
        }`